index.js

```diff
@@ -9,7 +9,7 @@ function convert(geojson) {
     var sourceId = hat();
 
     var style = {
-        version: 8,
+        version: 9,
```

Comment:
Oh man. It's now been zero days since mixing up style spec version with style versions.